docs/source/markdown/podman-network-create.1.md

```diff
@@ -16,7 +16,9 @@ If no options are provided, Podman assigns a free subnet and name for the networ
 Upon completion of creating the network, Podman displays the name of the newly added network.
 
 NOTE: The support for the network name "pasta" is deprecated and will be removed in the next major
```

Comment:
@Luap99 this might stick in my noggin better.  Sorry for the rehashing here, as EACK!, I think I suggested the current line in the first place.

```suggestion
NOTE: The support for "pasta" as a value passed in the *name* field is deprecated and will not be accepted in the next major
```

Reply:
Thanks, I reworded it a bit as I think using `field` is also confusing.